test/bigtest/interactors/piece-details-modal.js

```diff
@@ -29,4 +36,5 @@ export default interactor(class PieceDetailsModal {
   receiveButton = new Button('[data-test-receive-button]');
   cancelButton = new Button('[data-test-cancel-button]');
   barcodeInput = new BarcodeInput();
+  commentInput = new CommentInput();
```

Comment:
IMHO, this is shouldn't be here, the right place is under `piecesInLine`, so every comment input belongs to corresponding piece

Reply:
fixed